src/main/java/de/creelone/dismine/Dismine.java

```diff
@@ -111,8 +111,19 @@ public void onEnable() {
 		getCommand("speed").setExecutor(new SpeedCommand());
 		getCommand("flyspeed").setExecutor(new SpeedCommand());
 		getCommand("walkspeed").setExecutor(new SpeedCommand());
+		getCommand("fly").setExecutor(new FlyCommand());
+		getCommand("vanish").setExecutor(new VanishCommand());
+		getCommand("warp").setExecutor(new WarpCommand());
+		getCommand("setwarp").setExecutor(new WarpCommand());
+		getCommand("delwarp").setExecutor(new WarpCommand());
+		getCommand("home").setExecutor(new HomeCommand());
+		getCommand("homes").setExecutor(new HomeCommand());
+		getCommand("sethome").setExecutor(new HomeCommand());
+		getCommand("delhome").setExecutor(new HomeCommand());
```

Comment:
can you please use subcommands (ideally w/ autocomplete):
- /warp to - teleport to warp point
- /warp set|add - add warp point
- /warp delete|remove - delete warp point
- /homes set|add - add home
- /homes delete|remove - delete home
- /homes tp|teleport - teleport to home
- /homes or /homes list - list homes
- /homes list \<player\> - list homes of player (needs perms) *optional*

/home - alias for /homes tp

Reply:
declined

Reply:
what do you mean declined?
this is a team effort, you can't just veto stuff and give no reasons as to why
we can discuss this...